include/bluetooth/bluetooth.h

```diff
@@ -470,6 +488,48 @@ int bt_le_scan_start(const struct bt_le_scan_param *param, bt_le_scan_cb_t cb);
  */
 int bt_le_scan_stop(void);
 
+/** @brief Add device (LE) to whitelist.
+ *
+ *  Add peer device LE address to the whitelist.
+ *
+ *  @note The Whitelist cannot be modified when a role if an LE role is using
```

Comment:
`whitelist`?
use when or if, not both :)

Reply:
Yes, that sentence got a bit messed up. :)